service/frontend/workflowHandler.go

```diff
@@ -2846,6 +2877,7 @@ func (wh *WorkflowHandler) getHistory(scope int, domainID string, execution gen.
 
 	if len(historyEvents) > 0 {
 		wh.metricsClient.RecordTimer(scope, metrics.HistorySize, time.Duration(size))
+		domainScope.RecordTimer(metrics.HistorySize, time.Duration(size))
```

Comment:
Is there a reason we need to dual emit?  Can we just have single metric emitted with domainScope?

Reply:
Yes, dual emission is actually necessary in the longer term too, specifically for timer style metrics - as they cannot be aggregated between series. If we want to see a distribution of history size across all domains we will need to view the metrics with no domain tag. Alternately we could emit to a domain tag all too.